src/components/Forms/EditEvent.js

```diff
@@ -118,6 +165,37 @@ export default function EditEventForm(props) {
                             onChange={change.bind(null, "elocation")}
                         />
                     </Grid>
+                    <Grid item xs={12} sm={4}>
+                        <TextField
+                            id="longitude"
+                            label="Longitude"
+                            fullWidth
+                            helperText={textFieldError("longitude")}
+                            error={!!textFieldError("longitude")}
+                            value={longitude}
+                            onChange={change.bind(null, "longitude")} />
+                    </Grid>
+                    <Grid item xs={12} sm={4} style={{ display: 'flex' }}>
```

Comment:
why flex?

Reply:
this is the result if i dont use flex
![image](https://user-images.githubusercontent.com/45786074/83955550-d1209680-a808-11ea-85b4-08985ecc320e.png)